app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -2043,4 +2099,9 @@ export class Dispatcher {
       branchToCheckout
     )
   }
+
+  /** Record when the user takes no action on the stash entry */
+  public recordNoActionTakenOnStash(): Promise<void> {
```

Comment:
Since we didn't end up implementing the `x` button to close the diff view, I am using the toggle button in the `ChangesList` component as an alternative.